### PR TITLE
Add docs for all S components

### PR DESCRIPTION
**Pull request by ethanshar**
## Description
Add docs for S components

## Changelog
Add docs for S components

### CODE REVIEW

File: src/components/sectionsWheelPicker/sectionsWheelPicker.api.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
+{
+  "name": "SectionsWheelPicker",
+  "category": "form",
+  "description": "SectionsWheelPicker component for presenting set of wheelPickers",
```
Comment (Inbal-Tish) [inline src/components/sectionsWheelPicker/sectionsWheelPicker.api.json:4]:
```suggestion
  "description": "SectionsWheelPicker component for presenting set of WheelPickers",
```

File: src/components/segmentedControl/segmentedControl.api.json [hunk 2]
```diff
@@ -0,0 +1,23 @@
+{
+  "name": "SegmentedControl",
+  "category": "form",
+  "description": "SegmentedControl component for toggling two values or more",
+  "example": "https://github.com/wix/react-native-ui-lib/blob/master/demo/src/screens/componentScreens/SegmentedControlScreen.tsx",
+  "images": [],
```
Comment (Inbal-Tish) [inline src/components/segmentedControl/segmentedControl.api.json:6]: remove empty key

File: src/components/skeletonView/skeletonView.api.json [hunk 3]
```diff
@@ -0,0 +1,48 @@
+{
+  "name": "SkeletonView",
+  "category": "layoutsAndTemplates",
+  "description": "Allows showing a temporary skeleton view while your real view is loading",
+  "note": "View requires installing the 'react-native-shimmer-placeholder' and 'react-native-linear-gradient' library",
```
Comment (Inbal-Tish) [inline src/components/skeletonView/skeletonView.api.json:5]:
```suggestion
  "note": "Requires installing the 'react-native-shimmer-placeholder' and 'react-native-linear-gradient' libraries",
```

File: src/components/stackAggregator/stackAggregator.api.json [hunk 4]
```diff
@@ -0,0 +1,28 @@
+{
+  "name": "StackAggregator",
+  "category": "basic",
+  "description": "Stack aggregator component",
+  "modifiers": ["margin", "padding"],
+  "example": "https://github.com/wix/react-native-ui-lib/blob/master/demo/src/screens/componentScreens/StackAggregatorScreen.tsx",
+  "images": [],
```
Comment (Inbal-Tish) [inline src/components/stackAggregator/stackAggregator.api.json:7]: empty key

File: src/components/stepper/stepper.api.json [hunk 5]
```diff
@@ -0,0 +1,27 @@
+{
+  "name": "Stepper",
+  "category": "Form",
+  "description": "A stepper component",
+  "example": "https://github.com/wix/react-native-ui-lib/blob/master/demo/src/screens/componentScreens/StepperScreen.tsx",
+  "images": [],
```
Comment (Inbal-Tish) [inline src/components/stepper/stepper.api.json:6]: empty key

**Review by Inbal-Tish — COMMENTED** [review]
I don't see you added the prop's default value when exists...

File: src/components/segmentedControl/segmentedControl.api.json (hunk 2 above)
Comment (ethanshar) [reply src/components/segmentedControl/segmentedControl.api.json:6]: keeping it to remember adding images

File: src/components/stackAggregator/stackAggregator.api.json (hunk 4 above)
Comment (ethanshar) [reply src/components/stackAggregator/stackAggregator.api.json:7]: keeping it to remember adding images

File: src/components/stepper/stepper.api.json (hunk 5 above)
Comment (ethanshar) [reply src/components/stepper/stepper.api.json:6]: keeping it to remember adding images

**Review by Inbal-Tish — APPROVED** [review]

**Pull request merged**